lib/bike_brigade/importers/mailchimp_importer.ex

```diff
@@ -7,83 +7,77 @@ defmodule BikeBrigade.Importers.MailchimpImporter do
   alias BikeBrigade.Importers.Importer
   alias BikeBrigade.Riders
   alias BikeBrigade.Riders.Rider
+  alias BikeBrigade.Messaging.SlackWebhook
 
   @count 100
   @mailchimp "mailchimp"
   @import_failed_tag %{name: "Dispatch Import Failed", status: "active"}
 
-  def sync_existing_rider(%Rider{} = rider) do
-    with {:ok, account} <- Mailchimp.Account.get(),
-         {:ok, list} <- Mailchimp.Account.get_list(account, get_config(:list_id)),
-         {:ok, member} <- Mailchimp.List.get_member(list, rider.email),
-         {:ok, rider_attrs} <- build_rider(member) do
-      Riders.update_rider(rider, rider_attrs)
-    end
-  end
-
-  def sync_riders do
-    Multi.new()
-    |> Multi.run(:get_last_synced, fn repo, _changes ->
+  # TODO make this a singleton? / add locking
+  def sync_riders(opts \\ nil) do
```

Comment:
Added a trello card to properly mock Mailchimp without needing this https://trello.com/c/hL9bC1HX/93-create-a-fakemailchimp-to-allow-for-better-mocking-without-api-credentials